solr/core/src/test/org/apache/solr/cloud/TestRebalanceLeaders.java

```diff
@@ -558,14 +534,6 @@ private void concentratePropByRestartingJettys() throws Exception {
     // then start them again to concentrate the leaders. It's not necessary that all shards have a
     // leader.
 
-    ExecutorService executorService = ExecutorUtil.newMDCAwareCachedThreadPool("Start Jetty");
-
-    for (JettySolrRunner jetty : jettys) {
-      cluster.stopJettySolrRunner(jetty);
-    }
-
-    ExecutorUtil.shutdownAndAwaitTermination(executorService);
```

Comment:
Side change: the executor was started and stopped for no reasons. It is actually used right after.